AgueroMishel/oliverspace
# Adding Checkout Test Cases

**Scenario 1:** User is able to select Short-Term Plan
- Preconditions: No Plan is selected
- Test Steps: 
  1. Go to Home Page
  2. Click on Select Plan
  3. Select Short-Term Plan
- Expected: Short-Term Plan should be displayed on header

**Scenario 2:** User is able to select Location
- Preconditions: No Location is selected
- Test Steps: 
  1. Go to Home Page
  2. Click on Select Location
  3. Enter zip code
- Expected: Zip code should be displayed on header

**Scenario 3:** User is able to Checkout
- Preconditions: Plan is selected. Location is selected
- Test Steps: 
  1. Go to Furniture Page
  2. Add any item
  3. Open Cart
- Expected: Item should be displayed on Cart
